src/please.go

```diff
@@ -995,7 +999,23 @@ func main() {
 		defer pprof.WriteHeapProfile(f)
 	}
 
+	return command
+}
+
+
+
+func main() {
+	command := initBuild(os.Args)
+
+	runWatchedBuild = func(args []string) {
```

Comment:
I still don't think this should need to receive `args`. If you reuse the options you already have, all the flags will probably just get honoured automatically, no?